dashboard/src/components/AppList/AppListItem.tsx

```diff
@@ -2,39 +2,38 @@ import * as React from "react";
 import { Link } from "react-router-dom";
 
 import placeholder from "../../placeholder.png";
-import { IApp } from "../../shared/types";
+import { hapi } from "../../shared/hapi/release";
 import Card, { CardContent, CardIcon } from "../Card";
 import "../ChartList/ChartListItem.css";
 
 interface IAppListItemProps {
-  app: IApp;
+  app: hapi.release.Release;
 }
 
 class AppListItem extends React.Component<IAppListItemProps> {
   public render() {
     const { app } = this.props;
-    const release = app.data;
-    const icon = app.chart && app.chart.attributes.icon;
-    const iconSrc = icon ? `/api/chartsvc/${icon}` : placeholder;
-    const metadata = release.chart && release.chart.metadata;
+    const metadata = app.chart && app.chart.metadata;
+    const icon = metadata && metadata.icon;
```

Comment:
are we hotlinking to the icon from Chart.yaml? We did this in Monocular but I'm a bit concerned about doing this.

Reply:
yes, it's either that or contacting the `chartsvc` to retrieve the icon which I think would be slower?

Reply:
I had a chat with @migmartri about it and I think it's okay to hotlink for now (people are explicitly putting links into Chart.yamls so we can assume it's okay, in the future we may want to cache/process them somehow).